app/js/components/ArbitrationList.jsx

```diff
@@ -0,0 +1,68 @@
+/*global web3*/
+import React from 'react';
+import {Card, CardBody, CardHeader, CardTitle, Table, Alert} from 'reactstrap';
+import PropTypes from 'prop-types';
+import ArbitrationResult from "./ArbitrationResult";
+import {ARBITRATION_UNSOLVED} from "../features/arbitration/constants";
+
+function getArbitrationState(escrow) {
+  if(escrow.arbitration.open && escrow.arbitration.result === ARBITRATION_UNSOLVED){
+    return <p className="text-danger">In arbitration</p>;
+  }
+  return <p className="text-success">Arbitration completed</p>;
+}
+
+const Address = (props) => <span title={props.address}>{props.address.substring(0, 6) + "..." + props.address.substring(38)}</span>;
```

Comment:
There is a way to have an auto ellipsis I think in bootstrap. Here: https://getbootstrap.com/docs/4.2/utilities/text/#text-wrapping-and-overflow
It should work

Reply:
I made that function based on this comment here: https://medium.com/@lyricalpolymath/web3-design-principles-f21db2f240c1  
> show a compact versions of the hashes but always show the initial and end parts 
ie: 0xABCD…EFGH